src/stitches.config.ts

```diff
@@ -27,8 +29,14 @@ export const {
     },
     fontSizes: {
       1: '12px',
-      2: '13px',
-      3: '15px'
+      2: '14x',
```

Comment:
There is a typo in the 14 👀 
What if we move from px to rem?

Reply:
We can use both, I'll modify it in rem